include/h5-wrapper.hpp

```diff
@@ -154,7 +154,7 @@ class WriteHDF5Correlator {
       auto dset = file.createDataSet(h5_dataset_name, comp_type, dspace);
       dset.write(&corr[0], comp_type);
     } catch (H5::Exception &e) {
-      e.printError();
+      e.printErrorStack();
```

Comment:
Note: we might need some CMake magic to determine if this function exists rather than ```printError()```. It seems that different HDF5 versions have one but not the other...

Reply:
To pass the integration test one should actually use ```printError()```

Reply:
No, it depends on the version of the HDF5 library,... Newer versions do not have ```printError()```, olders do. Hence the comment for the ```rho_nf2``` branch to have its travis files updated.